docs/PrimeDAOErrors.md

```diff
@@ -0,0 +1,15 @@
+# Error Codes
```

Comment:
wondering if we can just use a .json file for that (maybe .jsonc (= json with comments)).
At least in test files we can read from that .json file instead of having to duplicate in 3 places?:
1. solidity
2. for tests
3. as markdown

Reply:
My idea for this markdown was that developers could look these up in the repo as needed, but you're right we reference the codes in the files multiple time. 

I think in the future I would love to implement something like Balancer has, where all the error messages are part of a Solidity Lib and all the `revert` functions are replaced with their own `_revert` function that formats the error code into a human readable message.
- [Example of revert in Balancer code](https://github.com/balancer-labs/balancer-v2-monorepo/blob/master/pkg/pool-weighted/contracts/lbp/LiquidityBootstrappingPool.sol#L372)
- [Error Lib in Balancer code](https://github.com/balancer-labs/balancer-v2-monorepo/blob/master/pkg/interfaces/contracts/solidity-utils/helpers/BalancerErrors.sol)